models/kendra.go

```diff
@@ -0,0 +1,65 @@
+package models
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/service/kendra"
+)
+
+var opts = kendra.Options{
+	Credentials: prov,
+	Region:      region,
+}
+
+var client = kendra.New(opts)
+
+type KendraResult struct {
+	ImgPath string
+	Title   string
+	Link    string
+	Excerpt string
+}
+
+type KendraResults struct {
+	QueryOutput kendra.QueryOutput
+	Results     []KendraResult
+	Query       string
+	Count       int
+}
+
+func queryOutputToResults(out kendra.QueryOutput) KendraResults {
+	results := KendraResults{
+		Results: []KendraResult{},
+	}
+
+	for _, item := range out.ResultItems {
+		res := KendraResult{
+			ImgPath: "images/placeholder.jpg",
+			Title:   *item.DocumentTitle.Text,
+			Excerpt: *item.DocumentExcerpt.Text,
+			Link:    *item.DocumentURI,
```

Comment:
not completely sure whats going on here, but the URI seems to be empty

Reply:
from the [docs](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/kendra@v1.54.6/types#QueryResultItem), it should be the original location of the document.

I hadnt verified it, just saw that it was a property that could be accessed from the query results. I dont think its horrible to keep, but ill defer to your preference here. If you're fine with keeping it there i'll add a TODO comment above it.

Reply:
hmm, im not sure why it doesn't work then

Reply:
ill remove it for the time being

Reply:
ok cool